coriolis/tests/api/v1/data/replicas_update_storage_mappings.yml

```diff
@@ -0,0 +1,112 @@
+
+- config:
+    original_storage_mappings:
+      backend_mappings:
+        - source: "source_1"
```

Comment:
I'd prefer to have inputs more similar to the actual API request body, so you can actually get the `new_storage_mappings` in the `expected_result`: https://github.com/cloudbase/coriolis/blob/master/coriolis/schemas/storage_mappings_schema.json

Therefore, please add `destination` key to each of the `backend_mappings` and `disk_mappings`:
```yaml
backend_mappings:
  - source: source_1
    destination: destination_1
  - source: source_2
    destination: destination_2
disk_mappings:
  - disk_id: disk_1
    destination: destination_1
  - disk_id: disk_2
    destination: destination_2
```
Now, you can set new storage mappings by changing the destination, expected result should contain the new destination mapping.